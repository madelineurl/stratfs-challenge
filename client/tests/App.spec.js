/* eslint-env jest */

import "@testing-library/jest-dom/extend-expect";
import { render, cleanup } from "@testing-library/react";
import React from "react";
import App from "../app";
// import axios from "axios";

afterEach(cleanup);

describe('App', () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });

  // it('renders a creditor name', () => {
  //   const { getByText } = render(<App />);
  //   expect(getByText('Creditor')).toBeInTheDocument();
  // });
});
