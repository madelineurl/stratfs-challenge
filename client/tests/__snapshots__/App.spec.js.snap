// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<DocumentFragment>
  <h3>
    There was a problem loading the data.
  </h3>
  <table>
    <tbody>
      <tr>
        <th>
          <input
            type="checkbox"
          />
        </th>
        <th>
          Creditor
        </th>
        <th>
          First Name
        </th>
        <th>
          Last Name
        </th>
        <th>
          Min Pay %
        </th>
        <th>
          Balance
        </th>
      </tr>
      <tr>
        <td />
        <td>
          <input
            name="creditorName"
          />
        </td>
        <td>
          <input
            name="firstName"
          />
        </td>
        <td>
          <input
            name="lastName"
          />
        </td>
        <td>
          <input
            name="minPaymentPercentage"
          />
        </td>
        <td>
          <input
            name="balance"
          />
        </td>
      </tr>
      <tr>
        <td>
          <button>
            Add Debt
          </button>
        </td>
        <td>
          <button>
            Remove Debt
          </button>
        </td>
      </tr>
      <tr>
        <td>
          Total Balance
        </td>
        <td>
          0
        </td>
      </tr>
      <tr>
        <td>
          Total Row Count: 0
        </td>
        <td>
          Check Row Count: 0
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
